### ISSUE TITLE
The code window is highlighted and blinking

### ISSUE THREAD

**Issue by 1921096212** [microsoft/vscode-cpptools#10328]
Type: <b>Bug</b>

Use binary search in plug-in Settings to eliminate the cause of the problem; When the.h file is opened, the highlighted code keeps flashing and cannot be read and used.

Extension version: 1.13.8
VS Code version: Code 1.74.2 (e8a3071ea4344d9d48ef8a4df2c097372b0c5161, 2022-12-20T10:29:14.590Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i5-12400 (12 x 2496)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.72GB (2.84GB free)|
|Process Argv|D:\\code\\AT32F435_437_Firmware_Library_V2.1.1\\utilities\\SEM\\at32f435_437_freertos_demo.code-workspace --crash-reporter-id ec86c5a7-d688-4ded-9638-22b2de9593ca|
|Screen Reader|no|
|VM|67%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30577860
nodejswelcome1cf:30587006
3biah626:30602489
3d0df643:30613357
89544117:30613380
fim-prod:30623723

```

</details>

**Comment by 1921096212** [comment]
An error message is displayed indicating that ms-vscode.cpptools belongs to the c/ C ++ plug-in

**Comment by sean-mcmanus** [comment]
It sounds like you're hitting https://github.com/microsoft/vscode-cpptools/issues/9689 . The fix should be available in our pending 1.14.0 (pre-release) update (maybe later this week).

Using setting "C_Cpp.autoAddFileAssociations": false is a workaround, but it'll make ".C" associating with C++ stop working if that's important.

**Comment by 1921096212** [comment]
> It sounds like you're hitting #9689 . The fix should be available in our pending 1.14.0 (pre-release) update (maybe later this week).
> 
> Using setting "C_Cpp.autoAddFileAssociations": false is a workaround, but it'll make ".C" associating with C++ stop working if that's important.



> It sounds like you're hitting #9689 . The fix should be available in our pending 1.14.0 (pre-release) update (maybe later this week).
> 
> Using setting "C_Cpp.autoAddFileAssociations": false is a workaround, but it'll make ".C" associating with C++ stop working if that's important.

ok

**Comment by sean-mcmanus** [comment]
The fix is available with 1.14.0 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.14.0
